data.json

```
@@ -143,6 +143,15 @@
     "dateAdded": "2018-03-11T16:47:14.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "NativeScript",
+    "difficulty": "easy",
+    "description": "Make your first contribution to the NativeScript community and win an awesome set of stickers. Among the stickers you will find a limited edition 'Contributor' sticker. ",
+    "reference": "https://www.nativescript.org/blog/nativescript-first-time-contribution-contest-continued-and-extended",
+    "image": "https://d2odgkulk9w7if.cloudfront.net/images/default-source/default-album/stickers-photo.jpg",
+    "dateAdded": "2019-01-25T08:36:39.000Z",
```

```suggestion
    "dateAdded": "2019-01-25T06:45:29.000Z",
```

Date comes from issue creation: https://github.com/swapagarwal/swag-for-dev/issues/257#issue-403026662